man/systemd.netdev.xml

```diff
@@ -1300,6 +1300,12 @@
           for delivery to the real destination. This option is mandatory.</para>
         </listitem>
         </varlistentry>
+      <varlistentry>
+        <term><varname>PeerPort=</varname></term>
+        <listitem>
+          <para>Specifies the peer port number. Defaults to unset. Note that when peer port is set <literal>Peer</literal> address is mandotory.</para>
```

Comment:
Please append `=` to the setting name:
`<literal>Peer</literal>` -> `<literal>Peer=</literal>`.